service/isiService.go

```diff
@@ -161,11 +162,23 @@ func (svc *isiService) ExportVolumeWithZone(ctx context.Context, isiPath, volNam
 	return exportID, nil
 }
 
-func (svc *isiService) CreateQuota(ctx context.Context, path, volName string, sizeInBytes int64, quotaEnabled bool) (string, error) {
+func (svc *isiService) CreateQuota(ctx context.Context, path, volName string, softLimit string, advisoryLimit string, softGracePrd string, sizeInBytes int64, quotaEnabled bool) (string, error) {
 	// Fetch log handler
 	log := utils.GetRunIDLogger(ctx)
-
 	log.Debugf("begin to create quota for '%s', size '%d', quota enabled: '%t'", volName, sizeInBytes, quotaEnabled)
+	var softi, advisoryi int64
+	var softlimitInt, advisoryLimitInt, softGracePrdInt int64
+	softGracePrdInt, _ = strconv.ParseInt(softGracePrd, 10, 64)
+
+	//converting soft limit from %ge to value
+	if softLimit != "" {
+		softi, _ = strconv.ParseInt(softLimit, 10, 64)
```

Comment:
if any error in the Parsing, please raise the error.

Reply:
addressed